modules/upgrade-cluster-version-parameters.adoc

```diff
@@ -0,0 +1,168 @@
+`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` hexadecimal value
+
+|`ClusterVersionSpec.desiredUpdate`
+|Optional. The cluster version that you want. If the current and intended
+version do not match, an upgrade starts. Set only a version from the list of
+recommended updates or your upgrade might fail.
+If an update exists with a version in the `availableUpdates` or history,
+you can specify the version field without specifying the image.
+
+If an upgrade fails, the Operator stops and reports the status of the failing
+component. Setting the `desiredUpdate` value to the previous version starts a
+rollback, but not all rollbacks succeed.
+
+|`ClusterVersionSpec.upstream`
+|Optional. Specify the preferred update server. If you do not specify a value,
+the CVO uses the best update server for the cluster and region.
+
+|`ClusterVersionSpec.channel`
+|Optional. Specify the channel to use to apply non-standard updates to the
+cluster.
+
+[IMPORTANT]
+====
+The default channel contains stable updates. Do not specify a
+`ClusterVersionSpec.channel` value on production clusters.
+====
+
+|`ClusterVersionSpec.overrides`
+|Optional. A list of overides for components that the CVO manages. Mark
+components as `unmanaged` to prevent the CVO from creating or updating the object.
+
+|`ClusterVersionStatus`
+|The status of available updates and any in-progress updates. The current field
+is set to the version that the cluster is reconciling to, and the conditions
+array reports whether the update succeeded, is in progress, or is failing.
+
+|`ClusterVersionStatus.desired`
+|The version that the cluster is reconciling towards. In clusters that have not
+fully initialized, this value displays the available image or tag version data.
+
+|`ClusterVersionStatus.history`
+|A list of the most recent versions applied to the cluster. This value might be
+empty during cluster initialization and is updated when a new update is applied.
+The list is ordered by date, where the newest update is first in the list.
+Updates in the history have state `Completed` if the rollout completed and
+`Partial` if the update failed or did not complete.
+
+|`ClusterVersionStatus.observedGeneration`
+|The version of the spec that is synchronized. If this value is not equal to
+the `metadata.generation` value, then the `desired` and `conditions` values
+represent a previous cluster version.
+
+|`ClusterVersionStatus.versionHash`
+|A representation of the content that the cluster uses during upgrade.
+You cannot set this parameter value.
+
+|`ClusterVersionStatus.conditions`
+|The condition of the CVO. Conditions are valid for the current `desiredUpdate`
+only when the `metadata.generation` value matches the
+`status.generation` value.
+
+* `Available` means that the upgrade to the `desiredUpdate` value completed.
+* `Progressing` means that an upgrade is in progress.
+* `Failing` means that an update is blocked by a temporary or permanent error.
```

Comment:
It will be useful to include information on structure of each condition. most important are `Reason`, `Message`
https://github.com/openshift/api/blob/fe0c0ad070ce15d30b0a6dd20adb0ef883efe11a/config/v1/types_cluster_operator.go#L110-L115